core-patch/src/test/scala/eu/neverblink/jelly/core/patch/helpers/PatchTestCases.scala

```diff
@@ -324,9 +324,9 @@ object PatchTestCases:
       rdfPatchRow(opt),
       rdfPatchRow(rdfPrefixEntry(0, "https://test.org/test/")),
       rdfPatchRow(rdfNameEntry(0, "")),
-      rdfPatchRowAdd(rdfPatchNamespace("test", rdfIri(1, 0)).setGDefaultGraph(rdfDefaultGraph())),
```

Comment:
Are you sure you don't need to add some new tests for this approach to encoding/decoding? If everything works on instanceof, are there no new possible edge cases?

Reply:
Nope, there are enough tests as it is. I can add some tests for when people misuse the low-level API and set the values to be garbage, but it will simply throw an exception so :shrug: I don't see that as an important thing to cover.